lib/support/wercker/bower_install_with_cache.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# HANDLES CACHE FOR BOWER DEPENDENCIES
```

Comment:
Maybe you want to start with `set -e`: http://www.davidpashley.com/articles/writing-robust-shell-scripts/